apps/platform-app/src/app/[locale]/(auth)/(components)/LinkedInLoginButton/LinkedInLoginButton.tsx

```diff
@@ -0,0 +1,34 @@
+'use client'
+import { AppRoutes } from '@/utils/routes'
+import { Button } from '@gdh/ui-system'
+import { LinkedIn } from '@gdh/ui-system/icons'
+import { signIn } from 'next-auth/react'
+import { useState } from 'react'
+import { type Roles } from '../../_models/User.model'
+import styles from '../GithubLoginButton/GithubLoginButton.module.scss'
+
+interface LinkedInLoginButtonProps {
+  label?: string
+  role: Roles.SPECIALIST | Roles.HUNTER
+}
+
+export const LinkedInLoginButton = (props: LinkedInLoginButtonProps) => {
```

Comment:
**praise**: Well-structured component for LinkedIn login, maintaining consistency with GitHub login.